# Introduction
This repository presents the implementation of a computational workflow designed to construct and evaluate a Profile Hidden Markov Model (HMM) focused on a specific protein domain of interest. The project involves generating a structure-based multiple sequence alignment of representative domain-containing proteins, from which an HMM is built and calibrated. The resulting model is tested against both positive (domain-containing) and negative (domain-absent) protein sequences to assess its classification performance. Additionally, a sequence-based HMM—built using an alignment generated by MUSCLE—is developed for comparative evaluation. This analysis aims to highlight the impact of structural information on the sensitivity and specificity of domain detection. The project was carried out as part of a Laboratory in Bioinformatics course during my MSc in Bioinformatics, and integrates principles of structural bioinformatics, statistical modeling, and sequence analysis.
